proxy/Dockerfile

```diff
@@ -1,9 +1,19 @@
 FROM haproxy:1.7-alpine
 MAINTAINER Autodesk <info@autodesk.com>
 
-RUN apk add --no-cache socat
-COPY haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg
-COPY errors/ /usr/local/etc/haproxy/errors/
+RUN apk add --no-cache \
+    socat \
+    supervisor \
+    rsyslog
+
+RUN sed -i '/imklog.so/d' /etc/rsyslog.conf
+
+COPY files_docker/etc/supervisord.conf /etc/supervisord.conf
+COPY files_docker/etc/rsyslog.d/haproxy.conf /etc/rsyslog.d/haproxy.conf
+COPY files_docker/usr/local/etc/haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg
```

Comment:
I would like to keep the haproxy.cfg and maintenance.http at the previous location